app/vlinsert/opentelemetry/opentelemetry_test.go

```diff
@@ -60,7 +80,7 @@ func TestPushProtoOk(t *testing.T) {
 				{
 					LogRecords: []pb.LogRecord{
 						{Attributes: []*pb.KeyValue{}, TimeUnixNano: 1234, SeverityNumber: 1, Body: pb.AnyValue{StringValue: ptrTo("log-line-message")}},
-						{Attributes: []*pb.KeyValue{}, TimeUnixNano: 1235, SeverityNumber: 21, Body: pb.AnyValue{StringValue: ptrTo("log-line-message-msg-2")}},
+						{Attributes: []*pb.KeyValue{}, TimeUnixNano: 1235, SeverityNumber: 25, Body: pb.AnyValue{StringValue: ptrTo("log-line-message-msg-2")}},
```

Comment:
Unspecified severity should be [zero](https://github.com/open-telemetry/opentelemetry-collector/blob/564818fd7f6291e18c1e1881d1596b72782199bd/pdata/internal/data/protogen/logs/v1/logs.pb.go#L37). I wonder why it was 21 and become 25?

Reply:
@makasim - I guess that this is overflow test. Added 4 (missed warns)

Reply:
For now 21 is valid fatal, which will break tested value